c2rust/src/bin/c2rust-transpile.rs

```diff
@@ -155,6 +155,10 @@ struct Args {
     /// Fail when the control-flow graph generates branching constructs
     #[clap(long)]
     fail_on_multiple: bool,
+
+    /// Derive `Debug` trait for any structs that allow it (i.e., do not recursively contain any unions)
+    #[clap(long)]
+    derive_debug: bool,
```

Comment:
Can we make the API for this `--derive`, where we give it a list of `trait`s to derive, which at this point are `Clone`, `Copy`, and `Debug`, and `Clone` and `Copy` are done by default?  So the `clap` API would probably be something like this:

```rust
use clap::ValueEnum;

#[derive(Clone, Copy, Debug, ValueEnum)]
enum Derive {
    Clone,
    Copy,
    Debug,
}

struct Args {
    ...
    /// `trait`s to automatically derive where possible
    #[clap(long)]
    derive: Vec<Derive>,
}
```

I'm not getting the `clap` derive API right, but something like that.